src/lib/viewers/media/DashViewer.js

```diff
@@ -345,6 +360,35 @@ class DashViewer extends VideoBaseViewer {
             );
         }
     }
+
+    /**
+     * Loads alternate audio streams
+     *
+     * @return {void}
+     */
+    loadAlternateAudio() {
+        const variants = this.player.getVariantTracks().sort((track1, track2) => track1.audioId - track2.audioId);
+        const audioIds = [];
+        const uniqueAudioVariants = [];
+        let i = 0;
+        for (i = 0; i < variants.length; i++) {
```

Comment:
Please space out your code more. I'd suggest a line break before `let i = 0;`, `this.audioTracks = `, and `if (this.audioTracks.length > 1) {`

Reply:
Done